config/routes.rb

```diff
@@ -1,56 +1,8 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
+    post '/login', to: "sessions#create"
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.